conf/application.conf

```diff
@@ -41,6 +41,11 @@ play.filters.enabled = [
   uk.gov.hmrc.play.bootstrap.filters.MDCFilter
 ]
 
+nunjucks {
+  optimizationLevel = -1
+  noCache = false
```

Comment:
There was an investigation about this config whether that cause a performance issue on prod or not.
Is that discussion finalised ?

Reply:
Seems to work OK in dev as per above. These settings have been merged into main and we can deploy to prod next week and observe if any issues. See PODS-7857.